src/main/java/io/leonis/torch/component/TextBackground.java

```diff
@@ -0,0 +1,58 @@
+package io.leonis.torch.component;
+
+import com.googlecode.lanterna.*;
+import com.googlecode.lanterna.graphics.*;
+import com.googlecode.lanterna.gui2.*;
+import java.awt.Color;
+import java.util.*;
+
+import java.util.stream.Stream;
+import lombok.*;
+
+/**
+ * The Class TextBackground
+ *
+ * This class is basically an {@link EmptySpace} with a text on top. The text is a tint brighter
+ * than the background color and will be centered based on the current size of the {@link
+ * com.googlecode.lanterna.screen.Screen}
+ *
+ * @author Jeroen de Jong
+ * @author Thomas Hakkers
+ */
+@Value
+@EqualsAndHashCode(callSuper = true)
+public final class TextBackground extends AbstractComponent<TextBackground> {
+
+  /**
+   * Background label
+   */
+  private final String text;
```

Comment:
Might be interesting to make `TextBackground` receive a Label / Component and rename it. It'd be more flexible and removes its dependency on `CenteredLabel`.